src/Facade.php

```diff
@@ -3,11 +3,37 @@
 
 class Facade
 {
-    use TestTrait;
+    use TestTrait {
```

Comment:
You can just do:
```php
use TestTrait {
        limitTo as public;
        minimumEvaluationRatio as public;
        shrinkingTimeLimit as public;
        withRand as public;
}
```
The methods below are not needed then.

But I think there's a reason they're protected, so why do you want to change that?

Reply:
Thanks! Didn't know that.

Have a look at https://github.com/wmde/eris/commit/7ba91f80a2481e1ec33734842160f1a2790c87b7#diff-9d046278d105ba90b2fecbbf9bc05c79
Before that all the methods on the trait were protected, because the only way to use them was to use the trait in PHPUnit test case. As soon as `Facade` was introduced some of the methods were made public to make it possible to call them from on `Facade`.
Theses were probably forgotten but they are needed to be able to use the library without hacking around.

Reply:
As an example of hacking around:
If you're using `Facade` the only way to set seed is this:
```php
$seed = 1504876177284329;
putenv("ERIS_SEED=$seed");
```
I don't think it is the proper way things should be :smile: